modules/issue_tracker/php/issuerowprovisioner.class.inc

```diff
@@ -99,7 +99,22 @@ class IssueRowProvisioner extends \LORIS\Data\Provisioners\DBRowProvisioner
             $module = &$modules[$mname];
         }
 
-        $cids = isset($row['centerId']) ? [$row['centerId']] : $this->_siteList;
+        if (isset($row['centerId'])) {
+            $cids = [intval($row['centerId'])];
+        } else {
+            $cids = $this->_siteList;
+        }
+
+        if (isset($row['centerId'])) {
+            $row['centerId'] = [$row['centerId']];
+        } else {
+            $row['centerId'] = array_map(
+                "strval",
+                \User::singleton()->getCenterIDs()
+            );
+            array_push($row['centerId'], "all");
+        }
```

Comment:
That's something we can discuss. Currently, in the data table, an issue with a NULL centerId has the value 'All Sites' in the Site column. The option 'all' is added to be able to filter such Issues. 

I was hesitant to replace the label All Sites with a comma separated list of all centerIDs (and convert Sites to a multiselect field is also a good idea). From the comments I received I feel like this option will be less confusing.

Reply:
Yeah, I think my main concern is that storing the value `all` in an array that's supposed to hold centerIDs is a bit confusing to me. Personally, I think going the route of comma separated sites and a multiselect would be better. This also becomes extensible in case we move in the direction of having the issues be associated with a subset of sites (i.e. multiple sites but not ALL sites).

Nut then again I'm not sure if a user would prefer to know that the issue is for "All Sites" vs. "A list of all sites". I guess in that case in the format column table there could be a cross check between the centerIds array and a queried list of sites from the back end, and if they match, the table could display "All Sites" instead of the comma separated list of sites.

Reply:
Plenty of good suggestions here... Thanks @HenriRabalais for your feedback.